### DISCUSSION TITLE
How to read the current feed rate and override ( $10 - Status report, mask )

### DISCUSSION THREAD

**Discussion by GrblGru** [grblHAL/core#206] [Q&A] [answered]
I have set $10 to 2  -> WPOS and and expect to get "FS:<feed rate>" and "Ov:<overrides>" in the realtime report.
But I always get only information like "<Jog|WPos:128,470,35,420,5,000,0,000,0,000|Bf:34,1023>"
Are there any more settings flags in $10 ? What do I have to do to send also the values for feed and override ? 

Thanks for your time to answer all our questions.

**Comment by terjeio** [comment, marked as answer]
> Are there any more settings flags in $10 ?

Yes - there are many. Use `$$=10` for details. More info [here](https://github.com/grblHAL/core/wiki/Additional-or-extended-settings) (I need to update this).

[ioSender](https://github.com/terjeio/ioSender) has an advanced settings UI you may try, the UI is built from data fetched from the controller:

![image](https://user-images.githubusercontent.com/20260062/196268183-4edffcc2-8352-41bd-b4ad-0f26de8022c4.png)

**Reply by GrblGru** [reply]
Great, thanks a lot
